substrate/client/informant/src/lib.rs

```diff
@@ -187,10 +187,12 @@ where
 				last_blocks.pop_front();
 			}
 
+			let best_indicator = if n.is_new_best { "🌟" } else { "✨" };
```

Comment:
```suggestion
			let best_indicator = if n.is_new_best { "🏆" } else { "🆕" };
```

As proposed by @koute